packages/manager/modules/manager-components/src/ssh-key-selector/index.js

```diff
@@ -0,0 +1,16 @@
+import angular from 'angular';
+
+import '@ovh-ux/ui-kit';
+
+import component from './ssh-key-selector.component';
+import service from './ssh-key-selector.service';
+
+const moduleName = 'ovhManagerComponentsSshKey';
+
+angular
+  .module(moduleName, ['oui'])
+  .component('sshKeySelector', component)
```

Comment:
Use prefix for naming the components.
```suggestion
  .component('ovhManagerSshKeySelector', component)
```